lib/broadway.ex

```diff
@@ -504,6 +504,14 @@ defmodule Broadway do
 
   Broadway currently exposes following Telemetry events:
 
+    * `[:broadway, :supervisor, :init]` - Dispatched when the supervision tree
```

Comment:
Just a nitpick: instead of `supervisor`, we could name this `topology` or `pipeline` for better proximity with naming from Broadway. WDYT?

Reply:
Great idea! I liked topology better and updated the PR with that one :D